quarto/04-QC-TTET01.qmd

```diff
@@ -0,0 +1,210 @@
+---
+title: "QCing Time-to-event Summary (TTET01)"
+editor: source
+format: 
+  html:
+    page-layout: full
+    code-fold: true
+    code-summary: "Show the code"
+    code-overflow: scroll 
+    df-print: kable
+---
+
+## Time-to-event Summary (TTET01) QC Workflow
+
+#### 1. Generate a table using {chevron}
+
+```{r}
+#| message: false
+#| code-summary: "Show the code"
+library(chevron)
```

Comment:
The renv is outdated. It is also using 4.3 instead 4.4. I think we should add a hidden check for versions at the beginning so to not compile anything if we do not have current version. It is useful to us to not merge outdated outputs